jujugui/static/gui/src/app/components/budget-table/row/row.js

```diff
@@ -409,7 +393,9 @@ class BudgetTableRow extends React.Component {
           <div>
             {this._generateSharedFields()}
             {this._generatePlanCols()}
-            {this._generateExtra()}
+            <div className="twelve-col no-margin-bottom">
```

Comment:
There are no layout divs in practice, only in the tests because populating the extra info is difficult :)